Source/Fuse.Text/Wrap.uno

```diff
@@ -42,9 +43,24 @@ namespace Fuse.Text
 			{
 				WrapLine(font, maxPixelWidth, ref minTolerance, ref maxTolerance, line, result);
 			}
+			if (maxLines > 0 && result.Count > maxLines)
+			{
+				result = result.Take(maxLines).ToList();
+				// adding `...`
+				InsertTruncationSringSymbol(font, result);
+				return result;
+			}
 			return result;
 		}
 
+		static void InsertTruncationSringSymbol(Font font, List<List<ShapedRun>> lines)
```

Comment:
Typo: Missing a `t` in `InsertTruncationSringSymbol`.

Reply:
oops! fixed.

Reply:
Thank you!